docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+- First, pair your HC04 (pin: 1234) or esp32 with your device or computer.  Look for a device which includes "mLRS" in its name.
+- iOS devices do not support the Bluetooth Serial Port Profile and QGroundControl does not support Bluetooth LE so QGroundControl will not connect over Bluetooth on iOS.  It has been reported that the SidePilot App works on iOS.
+- QGroundControl works well with Bluetooth on Android, Linux or Windows.  Create a new Bluetooth type connection via Application Settings -> Comm Links -> Add.
+- Mission Planner is recommended for Windows.  Connect to the com port associated with the Bluetooth connection after pairing.
+
+UDP or TCP over WiFi is another popular wireless option.
+- All three of the recommended Tx modules could be connected to an external esp32 board such as the M5 Stamp boards running the [Wireless Bridge](WIRELESS_BRIDGE.md) sketch.
+- All popular ground stations can connect via UDP
+- The standard UDP port for MAVLink is 14550.
+
+All three of the recommended Tx modules include serial ports which may be used to connect to your ground station.
+- Since PC serial ports are full RS232 ports which use higher voltage levels than the logic level serial connections on Tx modules, a USB ttl serial adapter is used to connect a Tx serial port to a PC or mobile device.
+- The FrSky R9M shares its single inverted serial port between CLI and MAVLink based on the position of [dip switch 1](FRSKY_R9.md#r9m-tx-module) at power on.  A [method](FRSKY_R9.md#diy-inverter-dongle) to invert the signals will be required.
+- The [MatekSys Tx modules](https://www.mateksys.com/?portfolio=mr24-30-tx#tab-id-2) provide 1 or 2 TTL level serial ports depending on the dip switch settings.
+
+Direct USB connection is another useful option for both PCs and mobile devices which support OTG USB.
+- Unfortunately this option is not available with the FrSky R9M.
+- This option does not work out of the box with the recommended MatekSys Tx modules for this use case because the USB port on these modules is blocked when installed in a radio JR bay.
+- It is possible to solder a type A USB cable directly to the USB pads on the back of the MatekSys Tx modules.
+- If you do this, you will need to flash the "siktelem" version of the firmware for your MatekSys Tx module.
+- If you solder a cable to these pads, you must then also use the soldered USB cable for flashing and not connect anything to the built-in type C jack.
+
+## 8. Radio calibration
+
+There are many calibration and setup steps required for ArduPilot on a new aircraft.  Here, we mention just the RC calibration step.  Not only because RC calibration needs to be repeated when switching to mLRS from a different RC system, but also because of a quirk you may encounter.
+
+- If you normally use QGroundControl as your ground station, be aware that it does not properly handle Radio calibration when using "rc_override".
```

Comment:
is there an intention with Radio camelcase?

Reply:
No, fixed